lib/contact_searcher.rb

```diff
@@ -107,6 +105,12 @@ def api_where(selector=nil)
             end
           when 'first_name', 'last_name'
             self.new_selector[k] = v.is_a?(String)? Regexp.new(v,Regexp::IGNORECASE) : v
+          when 'tags'
+            cs = Tag.find(v).map{|t| t.contact_ids}.flatten
```

Comment:
Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:contact_ids` as an argument to `map` instead of a block.